api/replication.storage/v1alpha1/volumegroupreplicationcontent_types.go

```diff
@@ -52,9 +56,8 @@ type VolumeGroupReplicationContentSpec struct {
 	// +optional
 	VolumeGroupReplicationClassName string `json:"volumeGroupReplicationClassName"`
 
-	// Source specifies whether the snapshot is (or should be) dynamically provisioned
+	// Source specifies whether the volume is (or should be) dynamically provisioned
```

Comment:
I guess that should be "volume group"?

Reply:
No, becuase we are storing the volume handles in the source of the VGRContent. So, I think it should be "volumes" only.

Reply:
Ah, righ. Can you include an example of a VGRContent in the docs?

Reply:
Sure, somehow I missed it _(though I had it in the back of my mind that I need to add that)_